crates/pop-cli/src/commands/up/parachain.rs

```diff
@@ -8,7 +8,7 @@ use cliclack::{
 };
 use console::{Emoji, Style, Term};
 use duct::cmd;
-use pop_common::sourcing::binary::Status;
+use pop_common::sourcing::Status;
```

Comment:
As suggested elsewhere, Status could/should be moved to root module as its fairly abstract to general purpose status reporting.